docs/core/components/data-column.component.md

```diff
@@ -467,6 +469,31 @@ const shouldPerformAction = this.columns
 if (shouldPerformAction) { /* action */}
 ```
 
+### Specifying max text length in the column
+
+If you want to restrict max number of characters displayed in the column you can use `truncated` param together with `maxTextLength`.
```

Comment:
Hmm... do we really need two properties for that functionality (truncated and maxTextLength)? I guess maxTextLength does nothing without setting truncated to true so maybe we should simplify it and keep only maxTextLength? When maxTextLength is set then text is truncated, when it's not set then there is no truncation

Reply:
Good idea, removed redundant one